tools/tsp-client/package.json

```
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "description": "A tool to generate Azure SDKs from TypeSpec",
   "main": "dist/index.js",
+  "homepage": "https://github.com/Azure/azure-sdk-tools",
```

```suggestion
  "homepage": "https://github.com/Azure/azure-sdk-tools",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Azure/azure-sdk-tools.git"
  },
```

It would be nice to also have the repository property

---

We had the same thought, I just pushed it :)

---

Also added a bugs link